api/python/provisioner/commands/configure_setup.py

```diff
@@ -90,199 +98,166 @@ def __attrs_post_init__(self):
+                input_data[key] = f'[{value}]'
+
+            elif key in keys_of_type_list:
+                input_data[key] = f'[\"{input_data[key]}\"]'
+
             else:
-                if input[key]:
-                    if input[key] == 'None':
-                        input[key] = '\"\"'
+                if input_data[key]:
+                    if input_data[key] == 'None':
+                        input_data[key] = '\"\"'
                     else:
-                        input[key] = f'\"{input[key]}\"'
+                        input_data[key] = f'\"{input_data[key]}\"'
                 else:
-                    input[key] = UNCHANGED
+                    input_data[key] = UNCHANGED
 
-    def _parse_pillar_key(self, key):
+    @staticmethod
+    def _parse_pillar_key(key):
         pillar_key = deepcopy(key)
         return pillar_key.replace(".", "/")
 
+    def _update_pillar_data(self, content, section, pillar_type):
+        """
+        Set pillar data.
+
+        Config content of default sections will be applied
+        across all nodes.
+
+        """
+        for pillar_key in content[section]:
+
+            key = f'{pillar_type}/{self._parse_pillar_key(pillar_key)}'
+            value_to_set = f"{content[section][pillar_key]}"
+
+            run_subprocess_cmd([
+                   "provisioner", "pillar_set",
+                   key, value_to_set])
+
+        if content.get('cluster', None):
+            if content.get('cluster').get('cluster_ip', None):
+                run_subprocess_cmd([
+                       "provisioner", "pillar_set",
+                       "s3clients/ip",
+                       f"{content.get('cluster').get('cluster_ip')}"])
+
+        logger.info(f"Data for '{section}' section updated.")
+
     def run(self, path, number_of_nodes):  # noqa: C901
 
+        logger.debug(
+            f"Number of nodes received to configure: {number_of_nodes}"
+        )
+        validate = ValidateSetup()
+
+        # TODO: Move to Path definition
         if not Path(path).is_file():
+            logger.error(
+               "config file is mandatory for setup configure. "
+               "Please provide a valid config file path. "
+            )
             raise ValueError('config file is missing')
 
         config = configparser.ConfigParser()
         config.read(path)
-        logger.info("Updating salt data :")
-        content = {section: dict(config.items(section)) for section in config.sections()}  # noqa: E501
-        logger.debug(f"params data {content}")
-
-        input_type = None
-        pillar_type = None
-        node_list = []
-        count = int(number_of_nodes)
-
-        for section in content:
-            input_type = section
-            pillar_type = section
+        logger.info("Updating salt data..")
+        content = {section: dict(config.items(section))
+                   for section in config.sections()}  # noqa: E501
```

Comment:
as long as you wrapped that long line `noqa` suppression comment is no more needed

Reply:
Yes, updated!